.github/workflows/push.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
     - master
-  pull_request_target:
+  pull_request:
```

Comment:
@vmarkovtsev how was the workflow here ?
Check that CI is green, and then revert before merge ?

Reply:
It is needed to build dependabot, so we can return back in the next PR.